src/packages/console/src/components/templateChooserModal.tsx

```diff
@@ -125,75 +52,76 @@ export const TemplateChooserModal = (props: ITemplateChooserModalProps): React.R
-                return (
-                  <ListItem
-                    key={templateCategory.templateCategoryId}
-                    button={true}
-                    selected={selectedTemplateCategoryId === templateCategory.templateCategoryId}
-                    onClick={(): void => onTemplateCategoryClicked(templateCategory)}
-                  >
-                    <ListItemText primary={templateCategory.name} />
-                  </ListItem>
-                );
-              })}
-            </List>
+          <Stack direction={Direction.Horizontal} isFullWidth={false}>
+            <Box maxWidth='300px'>
+              <List>
+                {templateCategories.map((templateCategory: TemplateCategory): React.ReactElement => {
+                  return (
+                    <ListItem
+                      key={templateCategory.templateCategoryId}
+                      button={true}
+                      selected={selectedTemplateCategoryId === templateCategory.templateCategoryId}
+                      onClick={(): void => onTemplateCategoryClicked(templateCategory)}
+                    >
+                      <ListItemText primary={templateCategory.name} />
+                    </ListItem>
+                  );
+                })}
+              </List>
+            </Box>
             {templates === null ? (
-              <Typography className={classes.errorMessage}>Failed to load templates. Please try again later.</Typography>
+              <Text>Failed to load templates. Please try again later.</Text>
             ) : templates === undefined ? (
-              <CircularProgress className={classes.loadingSpinner} />
+              <CircularProgress />
             ) : (
-              <List className={classes.templateList}>
+              <List>
                 {templates.map((template: Template): React.ReactElement => {
                   return (
-                    <ListItem key={template.templateId} className={classes.templateListItem} divider={true}>
-                      <ListItemAvatar>
-                        <img className={classes.templateImage} src={template.imageUrl} />
-                      </ListItemAvatar>
-                      <div className={classes.templateContent}>
-                        <ListItemText
-                          primary={template.name}
-                          secondary={template.description}
-                        />
-                        <div className={classes.templateButtons}>
+                    <ListItem key={template.templateId} divider={true}>
+                      <Stack direction={Direction.Horizontal} childAlignment={Alignment.Center} shouldAddGutters={true} defaultGutter={PaddingSize.Wide}>
+                        <ListItemAvatar>
+                          <img width='100px' src={template.imageUrl} />
+                        </ListItemAvatar>
+                        <Box maxWidth='340px'>
```

Comment:
why is this size here?

Reply:
Without giving it a width, the text would overflow and I was able to contain the text in a box

Reply:
this doesnt seem right, can you try to replace the `ListItemText` with ui-react `Text` please (you will need to put two in a vertical stack fo the name and description separately)